boards/nxp/imx95_evk/board.cmake

```diff
@@ -3,3 +3,14 @@
 if (CONFIG_SOF AND CONFIG_BOARD_IMX95_EVK_MIMX9596_M7_DDR)
   board_set_rimage_target(imx95)
 endif()
+
+if(CONFIG_BOARD_NXP_SPSDK_IMAGE OR (DEFINED ENV{CONFIG_BOARD_NXP_SPSDK_IMAGE}
+  AND "$ENV{CONFIG_BOARD_NXP_SPSDK_IMAGE}" STREQUAL "y"))
```

Comment:
as above, `CONFIG_` is a zephyr Kconfig thing, it has no place in an environmental value, this makes no sense

Reply:
@hakehuang 
Any suggestion here?
Thanks.

Reply:
you can rename it. @yangbolu1991